installer/pom.xml

```diff
@@ -72,8 +70,8 @@
                 </plugin>
                 <plugin>
                     <artifactId>maven-enforcer-plugin</artifactId>
-                    <version>1.4.1</version>
-                </plugin>
+                    <version>3.2.1</version>
+                 </plugin>
```

Comment:
```suggestion
                    <version>1.4.1</version>
                </plugin>
```

Reply:
I don't think it's a good idea to go back to 1.4.1, 
This version 1.4.1 doesn't handle the message presentation very well, which makes it very irrelevant to use this utility.
Moreover, if we go back to 1.4.1 we will also have to change VIVO.

Is it really a blocker to use more up-to-date plug-ins?

Reply:
Most likely it will impose new dependencies on maven or other components. I wouldn't change maven enforcer version in this PR.